plugin.video.hbogoeu/addon.xml

```diff
@@ -0,0 +1,71 @@
+<addon id="plugin.video.hbogoeu" name="HBO GO EU" provider-name="arvvoid" version="2.1.0">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0" />
+    <import addon="script.module.kodi-six" version="0.1.0" />
+    <import addon="script.module.requests" version="2.12.4" />
+    <import addon="script.module.pycryptodome" version="3.4.3" />
+    <import addon="script.module.defusedxml" version="0.6.0" />
+    <import addon="inputstream.adaptive" version="2.3.18" />
+    <import addon="script.module.inputstreamhelper" version="0.4.2" />
+  </requires>
+  <extension library="main.py" point="xbmc.python.pluginsource">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">HBO GO EU</summary>
+    <description lang="en_GB">
+Simple, Kodi add-on to access HBO® Go EU content. (This add-on is not officially commissioned/supported by HBO®.)
+
+Important, HBO® Go must be paid for!!!  You need a valid account!
+Register on the official HBO® Go website for your region.
+
+Curently support: Bosnia and Herzegovina, Bulgaria, Croatia, Czech Republic, Hungary, Macedonia, Montenegro, Polonia, Portugal, Romania, Serbia, Slovakia, Slovenija, Spain, Norway, Denmark, Sweden, Finland
+
+To report bugs or request assistence with the add-on go to: https://github.com/arvvoid/plugin.video.hbogoeu
+    </description>
+    <disclaimer lang="en_GB">
+This add-on is not officially commissioned/supported by HBO®. The trademark HBO® Go is registered by Home Box Office, Inc. For more information visit the official HBO® Go website for your region.
+I do not own any of the content this add-on lists. The content belongs to Home Box Office, Inc. All this add-on does is make simple HTTP requests to Hbo go servers to retrieve content just like any browser like Chrome, Firefox etc. would do!
+Playback is handled through inputstream.adaptive that use widevine to handle the DRM content like any web browser.
+THERE IS NO WARRANTY FOR THE ADD-ON, IT CAN BREAK AND STOP WORKING AT ANY TIME.
+If an official app is available for your platform, use it instead of this.
+Important: HBO® Go must be paid for!!! You need a valid HBO® Go account for the add-on to work! Register on the official HBO® Go website for your region
+    </disclaimer>
+    <language>en</language>
+    <platform>all</platform>
+    <license>GPL-2.0</license>
+    <forum>https://forum.kodi.tv/showthread.php?tid=339798</forum>
+    <source>https://github.com/arvvoid/plugin.video.hbogoeu</source>
+    <website>https://arvvoid.github.io/plugin.video.hbogoeu</website>
+    <news>
+v.2.1.0
```

Comment:
can you keep this only for the last version please?

Reply:
Fixed tnx